src/controllers/expenses.js

```diff
@@ -0,0 +1,143 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  filterByData,
+  filterByCategory,
+  filterById,
+  getExpenseById,
+  createExpense,
+  removeExpense,
+  updateExpense,
+} = require('../services/expenses.js');
+const userService = require('../services/users.js');
+const { allUsers } = require('../controllers/users.js');
+
+let expenses = [];
```

Comment:
Expenses array and method that clean expenses `clearExpensesArray` must be in services
This also applies to users